src/steps/os/linux.rs

```diff
@@ -54,7 +54,7 @@ impl Distribution {
             Some("clear-linux-os") => Distribution::ClearLinux,
             Some("fedora") | Some("nobara") => {
                 return if let Some(variant) = variant {
-                    if variant.contains(&"Silverblue") {
+                    if variant.contains(&"Silverblue") || variant.contains(&"Kinoite") {
                         Ok(Distribution::FedoraSilverblue)
```

Comment:
What about we rename `Distribution::FedoraSilverblue` to `Distribution::FedoraImmutable` since they only differ in their desktop environments

Reply:
If you want to add tests on parsing their `/etc/os-release` files, you can find them here: https://github.com/which-distro/os-release/tree/main/fedora

Reply:
I like it.  I'll have another commit up soon.